lib/matplotlib/collections.py

```diff
@@ -394,7 +394,7 @@ def set_hatch(self, hatch):
         *hatch* can be one of::
 
           /   - diagonal hatching
-          \   - back diagonal
+          \\   - back diagonal
```

Comment:
Actually I managed to build the docs, and the `\` in `Accept` below needed to be escaped more.
That should be fixed.